Sources/SwiftKotlinFramework/Utilities.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  Utilities.swift
+//  SwiftKotlinFramework
+//
+//  Created by Tor Langballe 07/07/2018.
+//
+
+import AST
+import Transform
+
+func firstCharIsUpper(str:String) -> Bool {
+    let s = String(str.first!)
+    return s == s.uppercased()
+}
+
+func changedValueToken(_ t:Token, _ val:String) -> Token {
+    if let origin = t.origin, let node = t.node {
+        return origin.newToken(t.kind, val, node)
+    }
+    return t
+}
```

Comment:
I would prefer it written as a `oldToken.replacing(value: newValue)`. It looks more swifty